politeiawww/api/www/v1/v1.go

```diff
@@ -1360,3 +1371,25 @@ type WSSubscribe struct {
 type WSPing struct {
 	Timestamp int64 `json:"timestamp"` // Server side timestamp
 }
+
+// SetTOTP attempts to set a TOTP key for the chosen TOTP type (Basic/UFI2 etc).
+type SetTOTP struct {
+	Type            TOTPTypeT `json:"type"`
+	CurrentTOTPCode string    `json:"currenttotpcode"`
```

Comment:
Why the naming difference between `SetTOTP.CurrentTOTPCode` and `VerifyTOTP.Code`?  Seems like `Code` is the more succinct choice.